utils.js

```diff
@@ -0,0 +1,13 @@
+const db = require("./db");
+
+exports.checkIfReviewExists = (id) => {
+  return db.query("SELECT * FROM reviews").then((data) => {
+    const count = 0;
+    const filteredArray = data.rows.filter((review) => {
```

Comment:
could also just check the length of result.rows from the select query -> if it's zero then we have proof that the given id doesn't exist